apis/hive/v1/azure/disk.go

```diff
@@ -0,0 +1,48 @@
+package azure
```

Comment:
I think we've copied these clients in because we have extra bits that wouldn't live in the installer client like functions for restarting instances. If we began consuming the installer clients then we would have to start there with changes, correct?

Reply:
I thought this was related to the client. Disregard.

Reply:
Per Meet™: This is the pattern all over the place. We should look into using the installer's objects instead of copying our own. But let's make a tech debt card for that and do it later.

Reply:
HIVE-1936